include/seqan3/alignment/configuration/utility.hpp

```diff
@@ -67,8 +67,16 @@ enum struct id : uint8_t
 {
     //!\brief Identifier for gap configuration.
     gap,
+    //!\brief Identifier for band configuration.
+    band,
+    //!\brief Identifier for free ends configuration.
+    sequence_ends,
+    //!\brief Identifier for score configuration.
+    score,
     //!\brief Identifier for global alignment configuration.
-    global
+    global,
+    //!\brief Identifier for output configuration.
+    output
```

Comment:
This is not part of the PR, right?

Reply:
True, but I made one commit (fc8e3f) that already includes all config ids. So I can base all configs on that commit, such that we can individually merge them later without conflicts in this file.

Reply:
Could you remove the band identifier from this commit, and then rebase all PRs?
The band won't be there in a while.